PDFNetPython/PDFNetPython.i

```diff
@@ -225,6 +225,10 @@
     #include "SDF/UndoManager.h"
     #include "SDF/ResultSnapshot.h"
     #include "SDF/DocSnapshot.h"
+	
+	// header files in /PDFNetC/Headers/Layout
+	#include "Layout/FlowDocument.h"
+	#include "Layout/Paragraph.h"
```

Comment:
This is just nitpicking, but indentation looks a bit off.

Reply:
Fixed